### PR TITLE
[Backend] Create randomly generated locations

### PR DESCRIPTION
**Pull request by gersbach**
## Summary
What does this PR contribute?

A feature for auto-populating locations if there is no location specified as well as adding any location to an already established tree.

## Test Plan
How should the person reviewing this PR test it out? For example, what screens in the app should they go to?
Simply examine the Alemany Farm, and create a tree. This new tree should be in the area of Alemany Farm

## Notes
(You can leave this empty if you don't have anything.)

## Screenshots and Tests
IMG_0573

If your feature is visual, attach before/after screenshots and ensure that tests still pass

### CODE REVIEW

**Review by ken-mar — COMMENTED** [review]
After reviewing the code I did confirm that most corrections can be made once ESLint changes are made. I believe the plug-in is not turned on. I tested the code to test the randomness of the tree placement and confirmed via firebase that the lat and long were randomly generated. The bounds must be adjusted so that tree placement can be generated within the bounds of the farm. This was out of the scope of the task and the developer is aware. The developer just needs to fix ESLint issues for approval.

File: src/database/firebase.js [hunk 1]
```diff
@@ -27,6 +27,48 @@ const userCollection = database.collection('users');
 const commentCollection = database.collection('comments');
 const additionalCollection = database.collection('additional');
 
+/**
+ * addTreeLocationTemp temporary function that adds a random location if the location is null
+ * imports Math and the map values from Default locations
```
Comment (kbrgl) [inline src/database/firebase.js:32]: This line is unnecessary, it's already evident what this code uses

Comment (GuessingMan) [reply src/database/firebase.js:32]: we should keep this comment in order to more easily delete this function at deployment

File: src/database/firebase.js [hunk 2]
```diff
@@ -72,33 +114,45 @@ export const getTree = async uuid => {
...
   } catch (e) {
     console.warn(e);
     throw e;
-    // TODO: Add error handling.
+    // TODO: Add error handling
   }
 };
 
 export const addTree = async tree => {
+  const treeCopy = addTreeLocationTemp(tree);
```
Comment (GuessingMan) [inline src/database/firebase.js:157]: add a comment that we should delete this at deployment

Comment (GuessingMan) [reply src/database/firebase.js:157]: added comment

**Review by GuessingMan — APPROVED** [review]

**Review by kbrgl — CHANGES_REQUESTED** [review]

**Pull request closed without merging**